recipe/meta.yaml

```diff
@@ -32,6 +32,8 @@ outputs:
         - pip
         - certifi >=2020.06.20
         - freetype >=2.3
```

Comment:
We can add a comment about freetype https://github.com/matplotlib/matplotlib/blob/v3.5.3/doc/devel/dependencies.rst#c-libraries